### PR TITLE
Kakaobrain Align model

### PR DESCRIPTION
**Pull request by jonasd4**
* Addition of public available reproduction of ALIGN model by kakaobrain (https://github.com/kakaobrain/coyo-align)
* This PR now also adds the `transformers` library as a dependency!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "torchtyping",
     "tqdm",
     "CLIP",
+    "transformers==4.40.1"
```
Comment (LukasMut) [inline setup.py:29]: @jonasd4 @a1247418 Should we keep `transformers` as a `thingsvision` dependency?

**Review by LukasMut — APPROVED** [review]
Small comment about the transformer library dependency. LGTM.

**Review by a1247418 — APPROVED** [review]
Looks good!

File: setup.py (hunk 1 above)
Comment (jonasd4) [reply setup.py:29]: I would vote for keeping it! Easier to add other models in the future and the people do not have to install something extra!

Comment (LukasMut) [reply setup.py:29]: @jonasd4 Let's change to "transformers==4.40.*"?

**Pull request merged**